packages/indexer-agent/src/query-fees/allocations.ts

```diff
@@ -132,10 +140,23 @@ export class AllocationReceiptCollector implements ReceiptCollector {
     receipts: AllocationReceipt[],
   ): Promise<void> {
     try {
-      // TODO: Encode the receipts batch
-      const encodedReceipts = '...'
+      // Encode the receipt batch to a buffer
+      // [allocationId, receipts[]] (in bytes)
+      const encodedReceipts = new BytesWriter(20 + (receipts.length * 112));
+      encodedReceipts.writeHex(receipts[0].allocation);
+      for (let receipt of receipts) {
+        // [fee, id, signature]
+        const fee = BigNumber.from(receipt.paymentAmount).toHexString();
+        const feePadding = 33 - (fee.length / 2);
+        encodedReceipts.writeZeroes(feePadding);
+        encodedReceipts.writeHex(fee);
+        encodedReceipts.writeHex(receipt.id);
+        encodedReceipts.writeHex(receipt.signature);
+      }
+
+      const clientUrl = 'TODO';
 
```

Comment:
I _think_ we don't need am extra URL here after constructing the Axios instance. But I may well remember its API wrong. Either way, can fix that after merging.